functions/New-DbaDbTable.ps1

```diff
@@ -376,11 +421,32 @@ function New-DbaDbTable {
                         }
                         $sqlcolumn = New-Object Microsoft.SqlServer.Management.Smo.Column $object, $column.Name, $dataType
                         $sqlcolumn.Nullable = $column.Nullable
+
+                        if ($column.Default) {
+                            if ($column.DefaultName) {
+                                $dfName = $column.DefaultName
+                            } else {
+                                $dfName = "DF_$name`_$($column.Name)"
```

Comment:
I'd go for a more readable  `"DF_$($name)_$($column.Name)"` . 
Not sure if we wanna check if the result is a correct name for a constraint (i.e. special characters, spaces, quoting, etc) or go straight for a (Get-Random)

Reply:
Was I too optimistic? 😅
What if we write on the help to avoid that kind of stuff and output the error if that happen?

Reply:
yeah, no problem in documenting the "limit"

Reply:
It seems that most of the characters are allowed (like with T-SQL). And it also adds escape characters when needed.

![image](https://user-images.githubusercontent.com/19521315/100555159-90727900-3291-11eb-8e9c-c747f49e73c2.png)

![image](https://user-images.githubusercontent.com/19521315/100555137-6de06000-3291-11eb-8893-4132ede9ede8.png)

If y'all agree, I suggest we go with this version

Reply:
hey @niphlod - do you agree? if so, i will merge 💯

Reply:
yup, merge it!